CHANGES.md

```diff
@@ -157,6 +157,11 @@
 
 ### New Modules
 
+  * `XMonad.Layout.CenteredIfSingle`
+
+    Layout modifier that, if only a single window is on screen, places that window
+    in the middle of the screen.
+
```

Comment:
This probably belongs elsewhere now?

Reply:
Yes, I missed that (I'll push a fix)

Reply:
Fixed by https://github.com/xmonad/xmonad-contrib/commit/0973107b290a9f4463f85264326990514ec63935

Reply:
thx :-)